InstantSearch.js/refresh-cache-user-action/src/app.js

```diff
@@ -28,7 +28,7 @@ search.addWidget(
     container: '#hits',
     templates: {
       item: `
-        <div class="ais-Hits-item">
+        <div>
           <header class="hit-name">
             {{{_highlightResult.name.value}}}
```

Comment:
We should migrate to the highlight function.

Reply:
Done in fe9075f230d59b1d1a9e1977a4e983bbdaa77fe6.